testplan/testing/multitest/entries/assertions.py

```diff
@@ -17,9 +17,10 @@
 import subprocess
 import sys
 import tempfile
-
 import lxml
 
+from ms.fix import FixMessage
```

Comment:
Let us catch up offline as this can be removed I think with similar conditionality that already exists.